lxd/main_cluster.go

```diff
@@ -235,6 +242,11 @@ func validateNewConfig(oldNodes []db.RaftNode, newNodes []db.RaftNode) error {
 			return fmt.Errorf("Changing cluster member ID is not supported")
 		}
 
+		// If the name field could not be populated, just ignore the new value.
+		if oldNode.Name != "" && newNode.Name != "" && oldNode.Name != newNode.Name {
```

Comment:
Just added the empty string check here so that the `name` field can be ignored in validation if we're dealing with a node whose `raft_nodes` table could not fill the `name` field.

Effectively, if the `name` field is blank, then it can be written to, and as well, a blank `name` can overwrite an existing name. This should be fine since once the nodes are back up and running, they should sort out the `name` field between themselves anyway.